db/connect.js

```diff
@@ -0,0 +1,7 @@
+const mongoose = require('mongoose')
+
+const connectDB = (url) => {
+    return mongoose.connect(url, { });
+};
```

Comment:
It's nice to have error logs and success message.

```suggestion
const connectDB = async (url) => {
  try {
    await mongoose.connect(url);
    console.log('MongoDB Connected');
  } catch (err) {
    console.error('Error connecting to MongoDB Atlas:', err);
  }
};
```

Reply:
change done on bd42f89